src/Controls/src/Core/Layout/Layout.cs

```diff
@@ -26,7 +26,33 @@ public abstract class Layout : View, Microsoft.Maui.ILayout, IList<IView>, IBind
 
 		public bool IsReadOnly => ((ICollection<IView>)_children).IsReadOnly;
 
-		public IView this[int index] { get => _children[index]; set => _children[index] = value; }
+		public IView this[int index]
```

Comment:
Should this be adding to and removing from the LayoutHandler?

Reply:
Fixed.